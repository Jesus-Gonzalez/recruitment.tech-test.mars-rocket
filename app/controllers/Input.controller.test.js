const { InputController } = require('./Input.controller');

describe('controllers', () => {
    describe('Input', () => {
        it('read: should match snapshot', () => {
            expect(InputController.read('input.test.dat')).toMatchSnapshot();
        });
    });
});
