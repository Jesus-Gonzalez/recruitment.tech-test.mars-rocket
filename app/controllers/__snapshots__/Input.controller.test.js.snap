// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`controllers Input read: should match snapshot 1`] = `
Object {
  "gridCoordinates": "4 4",
  "rocketConfiguration": Array [
    Object {
      "commandList": "RMLMRM",
      "coordinates": Object {
        "x": 0,
        "y": 1,
      },
      "orientation": "N",
    },
  ],
}
`;
